### PR TITLE
[RPA-833] Added users management empty subpage

### PR DESCRIPTION
**Pull request by Aeri4a**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

---

I agree to the terms of the RunBotics Contributor License Agreement.

### CODE REVIEW

**Review by Aeri4a — COMMENTED** [review]
Self review

File: runbotics/runbotics-orchestrator-ui/src/src-app/layouts/MainLayout/TopBar/Account.tsx [hunk 1]
```diff
@@ -109,6 +115,13 @@ const Account: FC = () => {
                 anchorEl={ref.current}
                 open={isOpen}
             >
+                <If condition={hasAdminAccess}>
+                    <MenuItem>
+                        <RouterLink href='/app/users' style={{ textDecoration: 'none', color: theme.palette.text.primary }}>
```
Comment (star-in-the-jar) [inline runbotics/runbotics-orchestrator-ui/src/src-app/layouts/MainLayout/TopBar/Account.tsx]: discuss with szymczakm

File: runbotics/runbotics-orchestrator-ui/src/src-app/views/users/UsersBrowseView/UsersBrowseView.tsx [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import React, { FunctionComponent } from 'react';
+
+const UsersBrowseView: FunctionComponent = () => {
+
+    return (
+        <>
+            Registration | Users
```
Comment (star-in-the-jar) [inline runbotics/runbotics-orchestrator-ui/src/src-app/views/users/UsersBrowseView/UsersBrowseView.tsx]: can you provide screenshot of this in the browser?

File: runbotics/runbotics-orchestrator-ui/src/src-app/views/users/UsersBrowseView/UsersBrowseView.tsx [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+        </>
+    );
+};
```
Comment (star-in-the-jar) [inline runbotics/runbotics-orchestrator-ui/src/src-app/views/users/UsersBrowseView/UsersBrowseView.tsx]: if there's nothing else but return in function body make it 

const UsersBrowseView: FunctionComponent = () => (
<>
            Registration | Users
</>
);

)

File: runbotics/runbotics-orchestrator-ui/src/src-app/views/users/UsersListView/UsersListView.tsx [hunk 4]
```diff
@@ -0,0 +1,12 @@
+import React, { FunctionComponent } from 'react';
+
+const UsersListView: FunctionComponent = () => {
+
+    return (
+        <>
+            Users
+        </>
+    );
+};
```
Comment (star-in-the-jar) [inline runbotics/runbotics-orchestrator-ui/src/src-app/views/users/UsersListView/UsersListView.tsx]: make it instant return () => (<>Users</>)

File: runbotics/runbotics-orchestrator-ui/src/src-app/views/users/UsersRegisterView/UsersRegisterView.tsx [hunk 5]
```diff
@@ -0,0 +1,12 @@
+import React, { FunctionComponent } from 'react';
+
+const UsersRegisterView: FunctionComponent = () => {
+
+    return (
+        <>
+            Register
+        </>
+    );
+};
```
Comment (star-in-the-jar) [inline runbotics/runbotics-orchestrator-ui/src/src-app/views/users/UsersRegisterView/UsersRegisterView.tsx]: as above ()=>(<>...</>)

**Review by star-in-the-jar — DISMISSED** [review]

**Review by preludium — DISMISSED** [review]

**Review by preludium — APPROVED** [review]

**Pull request merged**